man/git-fork.md

```diff
@@ -7,12 +7,18 @@ git-fork(1) -- Fork a repo on github
 
 ## DESCRIPTION
 
-  Fork the given github repo. Like clone but forks first.
+  If a github repo url is given, fork the repo. Like clone but forks first.
 
   1. forks the repo on github
   2. clones the repo into the current dir
   3. adds the original repo as a remote called upstream
 
+  If a url is not given and the current dir is a github repo, fork the repo.
+
+  1. forks the current repo
+  3. rename the 'origin' remote repo to 'upstream'
+  3. adds the forked repo as a remote called origin
```

Comment:
Either single quote every occurence of `upstream` and `origin`, or none. I can't decide which I think is best, but I'm positive that they should all be written the same.